source/api/commands/route.md

```diff
@@ -39,7 +39,11 @@ Supply a response `body` to *stub* in the matching route.
 
 **{% fa fa-angle-right %} method** ***(String)***
 
-Match the route to a specific method (`GET`, `POST`, `PUT`, etc). If no method is defined, Cypress will match `GET` requests by default.
+Match the route to a specific method (`GET`, `POST`, `PUT`, etc).
+
+{% note bolt %}
```

Comment:
😆 I just learned about `bolt` yesterday... using it plenty now 😄

Reply:
Yeah, it's a newer addition to notes, but pretty useful here.